app/models/schemas.py

```diff
@@ -473,7 +489,7 @@ def validate_payload_with_operation(self, data, **kwargs):
         if operation and payload:
             if operation in ["created", "updated"]:
                 OutboxCreateUpdatePayloadSchema().load(payload)
-            elif operation == "deleted":
+            elif operation in ["delete"]:
```

Comment:
No reason to use `in` if there's only one value
```suggestion
            elif operation == "delete":
```

Reply:
This change was not made, but I'll push a fix soon